src/test/java/seedu/mark/testutil/ReminderBuilder.java

```diff
@@ -0,0 +1,71 @@
+package seedu.mark.testutil;
+
+import static seedu.mark.testutil.TypicalBookmarks.ALICE;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import seedu.mark.model.bookmark.Bookmark;
+import seedu.mark.model.reminder.Note;
+import seedu.mark.model.reminder.Reminder;
+
+
+/**
+ * A utility class to help with building Reminder objects.
+ */
+public class ReminderBuilder {
+    public static final String DEFAULT_NOTE = "See Website";
```

Comment:
Use the static variable in the Note class if possible

Reply:
Okay